apps/earn-protocol/hooks/use-sdk.ts

```diff
@@ -79,7 +166,8 @@ export const useSDK = () => {
   )
 
   return {
-    getFleetClient,
+    getNewDepositTX,
```

Comment:
there is `getNewDepositTX` but `getWithdrawTX` :)

Reply:
Yes, the idea is that you either do a new deposit in the pool, in which you only need the poolID, or you could deposit more into the pool on an existing position. I think from the point of view of the protocol is the same, so I may scratch it later on, but for now I thought it could be useful. Withdraw does not make sense in a non-existing position, like deposit does